secretmanager/api/SecretManagerSample/Program.cs

```diff
@@ -218,22 +224,24 @@ public static void CreateSecret(string projectId, string secretId)
 
         // [START secretmanager_delete_secret]
         /// <summary>
-        /// Delete an existing secret with the given name.
+        /// Delete an existing secret with the given name and etag (optional).
         /// </summary>
         /// <param name="projectId">ID of the project where the secret resides.</param>
         /// <param name="secretId">ID of the secret.</param>
+	/// <param name="etag">Current secret etag.</param>
```

Comment:
Could we provide a quick explanation of what etags are/what they are used for?

Reply:
I think the comment that it should be the _current_ etag explains it quite sufficiently (as far as the code comment level of detail goes). But we have a dedicated doc that is more elaborate than what would fit on a single line here.